packages/flutter_tools/fuchsia_tools/README.md

```diff
@@ -0,0 +1,15 @@
+This directory serves as a placeholder directory for the package roots of
```

Comment:
Can `fuchsia_tools/` go under `bin/` instead of living at the top level? Additionally, can the Fuchsia entrypoints under `bin/` go under `bin/fuchsia_tools/` instead?

Reply:
I can move the fuchsia_tools directory to be in bin/ but we cannot move the entrypoints out of bin. The reason is that the //topaz/runtime/dart/flutter_test.gni has the path to the flutter tester hardcoded so changing them will break the fuchsia build.

Reply:
Kk. Let's just move the new directory under bin/ then. Thanks!

Reply:
Actually, it seems that our build rules make some assumptions about where the main file lives in relation to the package root. Moving the fuchsia_tools directory under bin/ causes the build to fail with the following error `Invalid package URI 'package:/fuchsia_asset_builder.dart'`